presto-product-tests/src/main/java/com/facebook/presto/tests/CreateDropViewTests.java

```diff
@@ -100,4 +107,24 @@ public void dropViewTest()
                     .failsWithMessage("does not exist");
         });
     }
+
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testViewOwnerPrivileges()
+    {
+        String viewName = "alice_view";
+        String tableName = "alice_table";
+        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
+
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        executeWith(createViewAs(viewName, format("SELECT * FROM %s", tableName), aliceExecutor), view -> {
+            assertThat(aliceExecutor.executeQuery(format("SHOW GRANTS ON %s", viewName)))
+                    .containsOnly(ImmutableList.of(
+                            row("alice", "hive", "default", viewName, "SELECT", Boolean.TRUE),
```

Comment:
At the least, we should test this manually for the release, and then for our next release configure our docker containers with SQL standard authoriation.